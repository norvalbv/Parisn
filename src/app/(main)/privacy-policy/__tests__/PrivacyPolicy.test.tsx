import React from 'react';
import renderer from 'react-test-renderer';

import PrivacyPolicy from '../page';

describe('<PrivacyPolicy />', () => {
  test('PrivacyPolicy props with mandatory props', () => {
    const tree = renderer.create(<PrivacyPolicy />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
