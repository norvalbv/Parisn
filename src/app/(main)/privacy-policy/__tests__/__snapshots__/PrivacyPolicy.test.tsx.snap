// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivacyPolicy /> PrivacyPolicy props with mandatory props 1`] = `
<div
  className="h-screen w-full grid place-items-center relative bg-primary-dark text-primary-light"
>
  <div
    className="w-1/2"
  >
    <p
      className="text-center"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolore illum eveniet error dolorem. Illo voluptate maxime nihil numquam molestiae aliquam, ullam, harum excepturi eligendi soluta laborum non, eos cumque porro dolore nostrum cum reprehenderit praesentium dolores pariatur quis. Nesciunt natus tempore qui temporibus quisquam dolores corrupti vel eaque consequuntur quo! Tenetur adipisci, fuga est harum officiis explicabo quidem maxime dolores deserunt perferendis doloremque! Tempore quis animi dicta totam ut quidem possimus dignissimos, suscipit iste deserunt voluptatem, a illum. Laboriosam quam provident sed molestiae quod omnis consequuntur, minima rem temporibus, eligendi ex deserunt doloribus. Quia eum ducimus a tenetur nemo! Corrupti voluptates ut, provident dicta perspiciatis quas labore, quos autem laboriosam repudiandae, reiciendis libero necessitatibus! Voluptatum sed officiis minus itaque, incidunt dolor velit odit laudantium debitis ex aut esse inventore maiores illo quia molestias explicabo adipisci expedita consequatur rerum. At qui magnam quo iusto mollitia ad recusandae esse accusamus rerum aperiam blanditiis nostrum sunt porro necessitatibus, explicabo temporibus quas, ducimus saepe totam consectetur obcaecati deserunt consequuntur? Facilis ea et, debitis error ad aut deserunt sunt. Dolore nobis, veritatis dolor laborum eaque iure tempore consequuntur suscipit voluptatibus dolorum quas cumque facilis enim maiores incidunt blanditiis minus. Rem voluptates aliquam ducimus sunt cum!
    </p>
  </div>
</div>
`;
